.github/workflows/release-go-cmd-oasis.yml

```diff
@@ -16,13 +16,7 @@ jobs:
     strategy:
       matrix:
         goos: [linux, windows, darwin]
-        goarch: ["386", amd64]
-        exclude:  
-          # windows/386 and darwin/386 seems useless 
-          - goarch: "386"
-            goos: windows 
-          - goarch: "386"
-            goos: darwin 
+        goarch: [amd64]
```

Comment:
Because we won't run `go` related tools on `386`, right?

Reply:
Because no one will need to use on `386`, it's too old platform.